src/lib/Backpack/devBackpack.cpp

```diff
@@ -90,12 +92,43 @@ void startPassthrough()
 }
 #endif
 
+#if defined(GPIO_PIN_BACKPACK_EN)
+static int debouncedRead(int pin) {
+    static const uint32_t debounce_delay = 5;
+    static const uint8_t max_samples = 5;
+    static const uint8_t min_matches = 2;
+
+    uint8_t matches = 0;
+    int last_state = digitalRead(pin);
+
+    for (uint8_t i = 0; i < max_samples; i++) {
+        delay(debounce_delay);
```

Comment:
We should not be calling delay in the main loop!
This would be better to read and increment a counter local to the file each time the check function is called and start the passthrough if the counter is over the stated value. It can be reset of the pin is high.

Reply:
Looks like a debounce delay of up to 1ms might be ok, and _should_ be enough to catch this.

The loop runs at about 200kHz.  So a counter with and initial trigger value of 100 (0.5ms) might be a good starting point.

Reply:
> We should not be calling delay in the main loop!
This would be better to read and increment a counter local to the file each time the check function is called and start the passthrough if the counter is over the stated value. It can be reset of the pin is high.

Interesting, I haven't considered debouncing across the loop. I thought the delay might be acceptable since we are only hitting it once we have a first suspicious low pin. But I'll rework it.

> Looks like a debounce delay of up to 1ms might be ok, and should be enough to catch this. 
> The loop runs at about 200kHz. So a counter with and initial trigger value of 100 (0.5ms) might be a good starting point.

I'll test it with my setup whether this is already enough.